Lecture5/Testing_Task1/TestInputProcessing.java

```diff
@@ -0,0 +1,71 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestInputProcessing {
+
+    private static BufferedReader numberReader;
+    private static BufferedReader operationReader;
+
+    static {
+        try {
+            numberReader = new BufferedReader(new FileReader(".\\Testing_Task1\\userInputNumber.txt"));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Printing stackstrace is not what you do expect as client when you have error msg. Message should be well formed and printed.

Reply:
fixed